uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/test/shared/controllers/leave-request-popup-controller_test.js

```diff
@@ -554,6 +577,105 @@
           });
         });
       });
+
+      describe('when absence period is changed', function () {
+        describe('for multiple days', function () {
+          describe('before from date is selected', function () {
+            it('should disable to date and to type', function () {
+              expect($ctrl.leaveRequest.from_date).toBeFalsy();
+            });
+          });
+
+          describe('after from date is selected', function () {
+            beforeEach(function () {
+              $ctrl.uiOptions.fromDate = new Date(date2017);
+              $ctrl.onDateChange($ctrl.uiOptions.fromDate, 'from');
+              $scope.$digest();
+            });
+
+            it('should enable to date and to type', function () {
+              expect($ctrl.leaveRequest.from_date).toBeTruthy();
```

Comment:
`vm.leaveRequest.to_date` and `vm.leaveRequest.to_date_type` should be checked not `from_date`